pkg/randchar/generator.go

```diff
@@ -244,6 +244,30 @@ func NewCharset(characters string) Charset {
 	}
 }
 
+// GetCharset returns the charset with the specified name.
+func GetCharset(charsetName string) Charset {
+	switch charsetName {
+	case "numeric":
+		return Numeric
+	case "lowercase":
+		return Lowercase
+	case "uppercase":
+		return Uppercase
+	case "letters":
+		return Letters
+	case "symbols":
+		return Symbols
+	case "alphanumeric":
+		return Alphanumeric
+	case "all":
+		return All
+	case "similar":
+		return Similar
+	default:
+		return Alphanumeric
```

Comment:
Do we want to use a default? This means we return `Alphanumeric` for every charset name that we do not know (yet). For example: `"numbers"`.

Other examples would include making a typo, e.g. `"symblos"`

This could result in hard-to-debug (or even undiscovered) bugs.

Do we want to return an error instead?

Reply:
returning `Charset, bool` would solve that I guess? No need for an error I'd say.

Reply:
Good point! I'll return `Charset, bool`.